app/javascript/packs/pattern_library.js

```diff
@@ -58,6 +60,7 @@ WebpackerReact.setup({ FormGroupRadioGroup })
 WebpackerReact.setup({ FormGroupTextArea })
 WebpackerReact.setup({ FormGroupTextInput })
 WebpackerReact.setup({ FormGroupTextValue })
+WebpackerReact.setup({ UnlabeledCheckbox })
```

Comment:
I think the checkbox component can be removed here
```suggestion
WebpackerReact.setup({ FormGroupTextValue })
```